tests/tkupchyn/homework_01/test_expressions.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+@pytest.mark.parametrize('string, expected_result', [('abc', 'a|b|c'),
+                                                     ('test', 't|e|s|t')])
+def test_string_join(string, expected_result):
+    result = str.join("|", string)
+    assert result == expected_result
+
+
+@pytest.mark.parametrize('words, expected_result', [(['abc', '', 'banana', 'ball'], 'banana'),
+                                                    (['test', '123123124'], '123123124')])
+def test_longest_word_positive(words, expected_result):
+    result = max(words, key=len)
+    assert result == expected_result
+
+
+@pytest.mark.parametrize('words', [([1, 3, "error", True])])
```

Comment:
no need in parametrize if there is only 1 case,


('words', [([1, 3, "error", True])])
its hard to read so many parenthesis combinations, try to use linebrakes as I suggested for clarity